docs/diagrams/ModelClassDiagram.puml

```diff
@@ -35,16 +37,18 @@ ModelManager -left-> "1" AddressBook
 ModelManager -right-> "1" UserPrefs
 UserPrefs .up.|> ReadOnlyUserPrefs
 
-AddressBook *--> "1" UniquePersonList
-UniquePersonList --> "~* all" Person
-Person *--> Name
-Person *--> Phone
-Person *--> Email
-Person *--> Address
-Person *--> "*" Tag
-
-Person -[hidden]up--> I
-UniquePersonList -[hidden]right-> I
+AddressBook *--> "1" UniqueStartupList
+UniqueStartupList --> "~* all" Person
```

Comment:
```suggestion
AddressBook *--> "1" UniqueStartupList
UniqueStartupList --> "~* all" Startup
```
Perhaps the classes above should also be changed to `Class UniqueStartupList` and `Class Startup`?